external/proto/darc.proto

```diff
@@ -7,32 +7,38 @@ option java_outer_classname = "DarcProto";
 // These are the messages used in the API-calls
 // ***
 
-// Darc is the basic structure representing an access control. A Darc can evolve in the way that
-// a new Darc points to the previous one and is signed by the owner(s) of the previous Darc.
+// Darc is the basic structure representing an access control. A Darc can
+// evolve in the way that a new Darc points to the previous one and is signed
+// by the owner(s) of the previous Darc.
 message Darc {
   // Version should be monotonically increasing over the evolution of a Darc.
   required uint64 version = 1;
-  // Description is a free-form field that can hold any data as required by the user.
-  // Darc itself will never depend on any of the data in here.
+  // Description is a free-form field that can hold any data as required by the
+  // user.  Darc itself will never depend on any of the data in here.
```

Comment:
Detail warning: can we have a consensus on whether to use one space or two spaces after a `.`? If both of you think two spaces are OK, so be it. But if it's half/half, it drives me crazy. I vote for one space only.

Reply:
I don't have a strong opinion on it, it's just my editor default since I don't manually reflow the text. I think @jeffallen does the same. I'm happy to change to one space. On a related note, we also have different text block widths, I have 80 characters but @jeffallen has something shorter, see the beginning of `darc.go` and the commend width for the difference. We also need to have an agreement on that.

Reply:
I type one space after period. My right margin was 70, and I've just fixed it to 80. Sorry for the noise.